### ISSUE TITLE
Add the Hubspot support widget to FlowForge Cloud

### ISSUE THREAD

**Issue by robmarcer** [flowforge/flowforge#1053]
### Description

As someone who wants to better understand our paying customers' feedback I want to encourage them to ask us for help by adding the Freshdesk support widget to Flowforge Cloud so that we can improve our docs and support.

This should increase feedback as the widgets tend to get higher customer interaction rates than mailer forms.

### Which customers would this be availble to

Other (See comments)

### Acceptance Criteria

- [ ] Add the support widget in to FlowForge Cloud

**Comment by robmarcer** [comment]
This is an example of the code to embed the widget.

```
<script>
	window.fwSettings={
	'widget_id':101000007442
	};
	!function(){if("function"!=typeof window.FreshworksWidget){var n=function(){n.q.push(arguments)};n.q=[],window.FreshworksWidget=n}}() 
</script>
<script type='text/javascript' src='https://euc-widget.freshworks.com/widgets/101000007442.js' async defer></script>
```

**Comment by knolleary** [comment]
We need to be mindful of how this will work for users running locally. As we have done with posthog etc, this would need to be configurable via `flowforge.yml` to get the instance specific details for FreshDesk.

We can then configure our managed instance of FlowForge with our account details - but users running locally won't be disturbed.

**Comment by robmarcer** [comment]
I wondered if we could address this with a more general purpose 'insert arbitrary javascript into all pages' feature rather than a FreshDesk specific feature? That might be more useful for Cloud, Enterprise and CE users as it would not be tied to one particular widget.

**Comment by robmarcer** [comment]
@ZJvandeWeg I don't think so as it's not been done, this is to add the support widget to FlowForge Cloud (or ideally as I said above allow anyone who's running FloweForge to add arbitrary JS to their installation).

**Comment by robmarcer** [comment]
@ZJvandeWeg @knolleary can we consider this for 1.3 please? I would suggest that rather than adding HubSpot specifically we allows admins of FF to add any JS they wish to their install. That would allow us to add Hubspot but would also provide value to FF admins (for example they could add Google Analytics).

**Comment by joepavitt** [comment]
I'm a little nervous about having the floating box every time someone logs into FlowForge, however I am very _pro_ the idea of making it easier to chat with support and ask questions.

How about an "Ask a Question" option in the user options (where we have Admin Settings, Documentation links atm) that, when clicked, opens the chat widget?

**Comment by joepavitt** [comment]
`'widget_id':101000007442` - what is the correct widget id we should be using @robmarcer?

**Comment by joepavitt** [comment]
Have a functional chat widget, although was hoping for us to be able to identify the user too (given that we have their log in details), but seems this is a premium feature: https://developers.hubspot.com/docs/api/conversation/visitor-identification

**Comment by joepavitt** [comment]
> I'm a little nervous about having the floating box every time someone logs into FlowForge, however I am very pro the idea of making it easier to chat with support and ask questions.

I changed my mind having seen the interactions we're getting through the Website. Went ahead with showing by default.

**Comment by joepavitt** [comment]
Closed by #1533
